estate/models/estate_model.py

```diff
@@ -0,0 +1,33 @@
+import datetime
+
+from odoo import fields, models
+
+
+class EstateModel(models.Model):
+    _name = "estate.property"
+    _description = "Estate description"
+    _property = "property"
+    name = fields.Char(required=True)
+    description = fields.Text()
+    postcode = fields.Char()
+    date_availability = fields.Date(copy=False, default=datetime.datetime.now() + datetime.timedelta(days=90))
```

Comment:
```suggestion
    date_availability = fields.Date(
        copy=False, 
        default=fields.Date(fields.Date.today(), months=3),
        string="Available from",
    )
```
1. Using datetime/relativedelta is not wrong (more or less, see next comment) but just so you know: there are helper that exists within Odoo for this, which avoids a new import.
2. Your field is a Date type, not Datetime, therefore using datetime.now() adds a cast that is easily avoidable.
3. We usually use the biggest unit possible: `days` -> `months`.
4. Also FYI, if not provided though the `string` parameter, Odoo deduce a label to display in the UI. In your case, it will deduce "Date Availability", which is not very user friendly 😉